.github/workflows/update-token.yml

```diff
@@ -0,0 +1,22 @@
+on:
+  workflow_dispatch:
+    inputs:
+      token:
+        description: "Slack token"
+        required: true
+        type: string
+
+jobs:
+  update-token:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - uses: actions/checkout@v3
+      - run: |
+          sed "s|\(https://join.slack.com/t/developersitalia/shared_invite/\)[a-zA-Z0-9_-]*|\1$TOKEN|" -i index.html
```

Comment:
This is most likely arbitrary code execution, do we care about it (it's just users with write access anyway) or should we filter the input? Maybe to be safe about possible changes in the future where the current assumption is no longer valid?

Reply:
Something like `"$TOKEN" =~ ^[[:alnum:]_-]{10,40}$`?

Reply:
Alternatively, we could avoid `sed` and rewrite the all `index.html` interpolating the token.

Reply:
@tensor5 right, I believe that rewriting `index.html` is cleaner